.eslintrc.json

```diff
@@ -7,6 +7,7 @@
         "space-before-function-paren": "off",
         "prefer-destructuring": "off",
         "object-shorthand": "off",
+        "react/prop-types": ["error", { "ignore": ["navigation"] }],
```

Comment:
FYI: Ignoring navigation props in prop-types is standard procedure, navigation props will be supplied to any view in our navigation stack, and we shouldn't be concerned with their structure, just their use.